src/geoip.h

```diff
@@ -12,10 +12,14 @@ class GeoDB
     ~GeoDB();
 
     std::string getGeoLocString(const char *ip_address) const;
+    std::string getGeoLocString(uint32_t ip_address) const;
```

Comment:
Yes the only reason I moved away from it was because the IPAddress interface changed - it means I would have to build those myself now, but that's possible.